test/Microsoft.Azure.Devices.Edge.Test/helpers/X509ManualProvisioningFixture.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Helpers
+{
+    using System;
+    using System.Security.Cryptography.X509Certificates;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Test.Common;
+    using Microsoft.Azure.Devices.Edge.Test.Common.Certs;
+    using NUnit.Framework;
+
+    public class X509ManualProvisioningFixture : ManualProvisioningFixture
+    {
+        public X509Thumbprint Thumbprint { get; set; }
+        public EdgeDevice Device { get; set; }
+
+        [OneTimeSetUp]
+        public async Task X509ProvisionEdgeAsync()
+        {
+            await Profiler.Run(
+                async () =>
+                {
+                    using (var cts = new CancellationTokenSource(Context.Current.SetupTimeout))
+                    {
+                        CancellationToken token = cts.Token;
+                        DateTime startTime = DateTime.Now;
+
+                        IdCertificates certs;
+                        (this.Thumbprint, certs) = await this.CreateDeviceIdCertAsync(Context.Current.DeviceId, token);
+
+                        EdgeDevice device = await EdgeDevice.GetOrCreateIdentityAsync(
```

Comment:
All the args to `EdgeDevice.GetOrCreateIdentityAsync` which follow this line are aligned to this line. Should they be indented one tab? E.g.:

```
EdgeDevice device = await EdgeDevice.GetOrCreateIdentityAsync(
    Context.Current.DeviceId + "-x509",
    this.iotHub,
    AuthenticationType.SelfSigned,
    this.Thumbprint,
    token);
```